### ISSUE TITLE
Modern Treasury SDK is outputting non-formated string on stdout

### ISSUE THREAD

**Issue by louis-jaris** [Modern-Treasury/modern-treasury-python#69]
The following python code will produce an output on the `stdout`:
```python3
import multiprocessing.pool
import logging

import modern_treasury

PRETTY_LOG_FORMAT = '%(asctime)s.%(msecs)03d [%(levelname)-8s] %(name)+25s - %(message)s'
logging.basicConfig(level="DEBUG", format=PRETTY_LOG_FORMAT, datefmt="%H:%M:%S")
log = logging.getLogger(__name__)


client = modern_treasury.ModernTreasury(
            organization_id="TODO-SET-ORGANIZATION-ID",
            api_key="TODO-SET-API-KEY",
            max_retries=6)


def simple_mt_call(_) -> None:
    log.debug("Doing a simple call to MT API with their SDK")
    client.ledgers.list()


if __name__ == '__main__':
    with multiprocessing.pool.ThreadPool(processes=100) as pool:
        pool.map(simple_mt_call, range(100))
```

When executing it, we get:
```bash
# Setup venv
python3 -m venv .venv; source .venv/bin/activate; pip install modern-treasury
# Run example above, redirect standard logs in a dedicated file
python example.py 2> stderr.log
Retry request in 0 seconds
Retry request in 0 seconds
Retry request in 0 seconds
Retry request in 0.0767289917805386 seconds
Retry request in 0.0964948210855312 seconds
Retry request in 0.1219694883958482 seconds
Retry request in 0 seconds
Retry request in 0.14873019160568857 seconds
```

----

Proposed change: instead of using `print` in this SDK, use a logger following python's best practices for logging inside a library.

Resources:
* https://docs.python.org/3/howto/logging.html
* https://docs.python.org/3/howto/logging-cookbook.html
* https://docs.python.org/3/library/logging.html

**Comment by rattrayalex** [comment]
Thank you for the report @louis-jaris ! We'll get that `print` call removed shortly, and hope to add proper logging soon.

**Comment by louis-jaris** [comment]
Thanks for the quick response @rattrayalex 🙏 

I see that it will be released in `1.10.0` (PR https://github.com/Modern-Treasury/modern-treasury-python/pull/71) 👌

I also see that you declared releases on GH (`1.9.0` for example https://github.com/Modern-Treasury/modern-treasury-python/releases), however, I do not see the latest release in PyPI: https://pypi.org/project/modern-treasury/. I can only get up to `1.6.0` there. Is that normal?

**Comment by cleb11** [comment]
@louis-jaris `1.10` has been released and this issue should be resolved!
